typescript/deploy-sdk/package.json

```diff
@@ -10,7 +10,8 @@
   ],
   "types": "./dist/index.d.ts",
   "exports": {
```

Comment:
Should we also add the type exports here like it is done from the radix-sdk for the `testing` sub import path?